.gitignore

```diff
@@ -9,6 +9,7 @@ coverage/
 web/client/libs/Cesium/
 web/cesium.zip
 web/.classpath
+web/overlays
```

Comment:
why this change?

Reply:
@agpenton ?

Reply:
nothing in particular, I just saw it new after I ran the environment, and since I didn't see it before I just ignored it.

Reply:
please remove it

Reply:
resolved here https://github.com/geosolutions-it/MapStore2/pull/6490/commits/e597002a1188e6809cdeabaaca0129fb396e4121